main.py

```diff
@@ -134,36 +132,70 @@ def shutdown(self):
         """
 
     @rest("/v2/evc/", methods=["GET"])
-    def list_circuits(self):
+    def list_circuits(self, request: Request) -> JSONResponse:
         """Endpoint to return circuits stored.
 
         archive query arg if defined (not null) will be filtered
         accordingly, by default only non archived evcs will be listed
         """
         log.debug("list_circuits /v2/evc")
-        args = request.args.to_dict()
-        archived = args.pop("archived", "false").lower()
+        args = request.query_params
+        archived = args.get("archived", "false").lower()
+        args = {k: v for k, v in args.items() if k not in {"archived"}}
         circuits = self.mongo_controller.get_circuits(archived=archived,
                                                       metadata=args)
         circuits = circuits['circuits']
-        return jsonify(circuits), 200
+        return JSONResponse(circuits)
 
-    @rest("/v2/evc/<circuit_id>", methods=["GET"])
-    def get_circuit(self, circuit_id):
+    @rest("/v2/evc/schedule", methods=["GET"])
```

Comment:
I had to move this route up, otherwise the routing order would match `GET /v2/evc/{circuit_id}`, it turns out that `starlette` won't automatically try to reorder so whoever is defining the routes should be aware of it. On `kytos` core as `@rest` routes are imported ordering will be guaranteed.